src/Education.UnitTests/RuleHelpXamlTranslatorTests.cs

```diff
@@ -147,11 +149,160 @@ public void TranslateHtmlToXaml_Span_AddsSpan()
             result.Should().Be(expectedText);
         }
 
-        private static IRuleHelpXamlTranslator CreateTestSubject(IXamlWriterFactory xamlWriterFactory = null)
+        [TestMethod]
```

Comment:
I am wondering now if it makes sense to test if the DiffTranslator implementation is correct considering that should be the task of the DiffTranslatorTests. Wouldn't it be okay to just test if you replace your marked text with whatever I return? 

eg. use a Mock<IDiffTranslator> and have it return random strings. Because now whoever makes future changes in the diff translator need to adjust DiffTranslatorTests and these tests. WDYT